sw/device/lib/testing/aes_testutils.c

```diff
@@ -0,0 +1,21 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/aes_testutils.h"
+
+#include "sw/device/lib/dif/dif_aes.h"
+#include "sw/device/lib/runtime/ibex.h"
+#include "sw/device/lib/testing/check.h"
+#include "sw/device/lib/testing/rand_testutils.h"
+
+static bool aes_testutils_get_status(dif_aes_t *aes, dif_aes_status_t flag) {
```

Comment:
the aes_idle test as described in the test plan needs this to execute faster than the current depth allows. Can you use inlining here so the overhead accrued here doesn't make it less feasible or edit the testplan so this isn't planned to be performed in such a short window?